arch/arm/src/stm32/Kconfig

```diff
@@ -2417,6 +2417,22 @@ config STM32_HAVE_IP_DMA_V2
 	bool
 	default n
 
+config STM32_HAVE_IP_SPI_V1
```

Comment:
These are data sheet dependent and invariant. Why not encapsulate it in the chip.h?  Leaving it up the the end user requires them to do work that was already done when an new SoC is added.